src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 40;
+
```

Comment:
here you can add variables for 20 & 50 days, also it's better to create variable for storing data of final amount: 
const total = days * 40;

Reply:
agree